examples/cloudwatch/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "name" {
```

Comment:
The module itself also has a variable "name", which defaults to `unset`. Not sure if this is required here.